documentation/slate/source/includes/realtime.md

```diff
@@ -0,0 +1,206 @@
+<a name="realtime"></a> Real time integration in Navita
+===============================================
+
+Several endpoints can integrate real time information in their responses. In the response received, a disruption will be present and additional information will be provided if the parameter `data_freshness` is set to `realtime`.
```

Comment:
Disruption is present even without 'data_freshness=realtime' for information. An other request has to be made to apply disruption.